package.json

```diff
@@ -58,6 +58,35 @@
             "Requirement"
           ]
         }
+      },
+      "settings": {
+        "region": {
+          "title": "Sentry server region",
+          "description": "The region your Sentry account is hosted in (e.g. US/EU)",
```

Comment:
Might want to check the language here, I think Sentry call this "Data Storage Location"?